src/main/java/caculator/domain/StringCalculator.java

```diff
@@ -0,0 +1,84 @@
+package caculator.domain;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class StringCalculator {
+
+    public static final int WRONG_RESULT = 0;
+    public static final int MINIMUM_FORMULA = 3;
+    public static final int EVEN = 2;
+    public static final int ODD_RESULT = 1;
+    public static final int FIRST_NUMBER = 0;
+    public static final int FIRST_SIGN = 1;
+    public static final int SECOND_NUMBER = 2;
+    public static final int NEXT_NUMBER = 1;
+
+
+    public int calculate(int numberOne, String sign, int numberTwo) {
+        if (isNotSign(sign)) {
+            throw new IllegalArgumentException("제대로된 사칙연산기호를 입력해 주세요. 입력 값: " + sign);
+        }
+
+        if (sign.equals("+")) {
+            return numberOne + numberTwo;
+        }
+        if (sign.equals("-")) {
+            return numberOne - numberTwo;
+        }
+        if (sign.equals("*")) {
+            return numberOne * numberTwo;
+        }
+        if (sign.equals("/")) {
+            return numberOne / numberTwo;
+        }
+        return WRONG_RESULT;
+    }
+
+    public int playCalculate(String inputString) {
+        if (isNull(inputString)) {
```

Comment:
```suggestion
        if (Objects.isNull(inputString)) {
```
Java에서 제공해 주는 기본 문법들을 최대한 활용해 보면 좋을것 같습니다.